lua/ulx/modules/sh/cfc_warn_after.lua

```diff
@@ -1,23 +1,26 @@
+CFCUlxCommands.warnAfter = CFCUlxCommands.warnAfter or {}
+local cmd = CFCUlxCommands.warnAfter
+
 local CATEGORY_NAME = "Utility"
 
-local function warnban( calling_ply, target_ply, minutes, reason )
+function cmd.warnBan( calling_ply, target_ply, minutes, reason )
     ulx.ban( calling_ply, target_ply, minutes, reason )
     local command = string.format( 'awarn_warn \"%s" "%s (%s ban)" ', target_ply:SteamID(), reason, ULib.secondsToStringTime( minutes*60 ) )
     calling_ply:ConCommand( command ) 
 end
-local ban = ulx.command( CATEGORY_NAME, "ulx warnban", warnban, "!warnban", false, false, true )
+local ban = ulx.command( CATEGORY_NAME, "ulx warnban", cmd.warnBan, "!warnban", false, false, true )
 ban:addParam{ type=ULib.cmds.PlayerArg }
```

Comment:
```suggestion
ban:addParam{ type = ULib.cmds.PlayerArg }
```
This isn't your job, and probably shouldn't be done in this PR, but otherwise it'll never happen so.
Noticed a bunch of these commands don't have spaces on their `=`s
If you'd go ahead and fix those for us, that'd be very much appreciated :)